app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -36,7 +36,7 @@ def info
     def mfa_configuration_count; end
 
     def mfa_configuration_description
-      return '' if !disabled?
+      return '' if mfa_configuration_count <= 0
```

Comment:
Similar to previous comment, `.blank?` or `.zero?` might be better options here, if we're expecting it to either be 0 or positive.

```suggestion
      return '' if mfa_configuration_count.zero?
```

Reply:
Changed that to `.zero?`